realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -79,6 +79,19 @@
  * A standard pattern for working with Realm in Android activities can be seen below:
  * <p>
  * <pre>
+ * public class RealmApplication extends Application {
+ *
+ *     private static RealmConfiguration defaultRealmConfig;
```

Comment:
Not really. It just complicates the example IMO:

```
public void onCreate() {
  RealmConfiguration realmConfig = new RealmConfiguration.Builder(this).build();
  Realm.setDefaultConfiguration(realmConfig);
}  
```

Would be enough IMO (at least for a basic intro, which I think this should be).